DigitalOffice/Tests/UserServiceTests/Repositories/UserRepositoryTests.cs

```diff
@@ -79,7 +80,27 @@ public void ShouldReturnUserIfUserWithRequiredIdExists()
             Assert.IsInstanceOf<DbUser>(resultUser);
             SerializerAssert.AreEqual(dbUser, resultUser);
         }
+        #endregion
 
+        #region GetUserByEmail
+        [Test]
+        public void ShouldThrowExceptionIfUserWithRequiredEmailDoesNotExist()
+        {
+            Assert.Throws<Exception>(() => repository.GetUserByEmail(string.Empty), ExceptionMessage);
```

Comment:
Make sure dbContext.Users hasn't been changed

Reply:
Wait, but how it can be changed? I just need to get the user, not to edit

Reply:
Is this check really needed?

Reply:
Yes. You are testing the behavior of the repository. In your scenario, you are getting something from the database. It is very important to check that the repository is working correctly with the database. For example, if the repository method not only getting something from the database, but also deletes the item, then this can lead to serious errors in the system.

Reply:
Fixed